source/Octopus.Server.Client/Repositories/Async/BasicRepository.cs

```diff
@@ -253,6 +253,26 @@ public async Task<List<TResource>> GetAll(CancellationToken cancellationToken)
             return await Client.Get<List<TResource>>(link, parameters, cancellationToken).ConfigureAwait(false);
         }
 
+        public Task<List<TResource>> FindByPartialName(string partialName, string path, object pathParameters, CancellationToken cancellationToken)
+        {
+            ThrowIfServerVersionIsNotCompatible(cancellationToken).ConfigureAwait(false);
+
+            partialName = (partialName ?? string.Empty).Trim();
+            if (pathParameters == null)
+                pathParameters = new { partialName = partialName};
```

Comment:
Minor nitpick: could use `??=` here

Reply:
I might keep this as-is for consistency. Also subjectively i find this easier to read :)